apps/api/v2/src/modules/slots/slots-2024-09-04/controllers/slots.controller.ts

```diff
@@ -173,6 +174,13 @@ export class SlotsController_2024_09_04 {
       2024-08-13 (will have hours 00:00:00 aka at very beginning of the date) or you can specify hours manually like 2024-08-13T09:00:00Z.`,
     example: "2050-09-05",
   })
+  @ApiQuery({
+    name: "rescheduleUid",
```

Comment:
I think we can improve naming here. `rescheduleUid` is not specific enough - the name should include what are we rescheduling specifically. Although longer, I think I would rename it as `bookingUidToReschedule`.

Do the same in [get-slots.input.ts](https://github.com/calcom/cal.com/pull/21626/files#diff-d5eeca66586516982c5f317b7c9e26284851e3bffd6032c51096d1e2c0c2392c) and in the [slots-input.service.ts](https://github.com/calcom/cal.com/pull/21626/files#diff-b13009a519718aba36fa51f007ce9a3b904b2bb1571ac45f57866f4bf06ea67a) have:
```
rescheduleUid: query.bookingUidToReschedule || null,
```